jetty-http/src/main/java/org/eclipse/jetty/http/QuotedQualityCSV.java

```diff
@@ -41,15 +42,20 @@
     private final List<String> _values = new ArrayList<>();
     private final List<Double> _quality = new ArrayList<>();
     private boolean _sorted = false;
+    private final Function<String, Integer> tieBreakerFunction;
```

Comment:
Oh ok, it is not a comparitor... more like a quality adjustor value.... hmm will think on the name.